lib/src/Base/Type/openturns/TriangularMatrix.hxx

```diff
@@ -129,11 +129,13 @@ public:
   /** IdentityMatrix multiplications  */
   TriangularMatrix operator * (const IdentityMatrix & m) const;
 
+#ifndef SWIG
```

Comment:
Didn't you say at the training session it would be cleaner to handle this in .i files?

Reply:
yes